dlp/system-test/metadata.test.js

```diff
@@ -17,17 +17,40 @@
 const {assert} = require('chai');
 const {describe, it, before} = require('mocha');
 const cp = require('child_process');
+const uuid = require('uuid');
 const DLP = require('@google-cloud/dlp');
 
+const dataProject = 'DATA_PROJECT';
+const dataSetId = 'DATASET_ID';
+const tableId = 'TABLE_ID';
+const fieldId = 'FIELD_ID';
+
```

Comment:
We can use the public dataset shared in the protocol example.

Reply:
Updated, but we still need `outputPath` before we can remove the it.skip() from test cases

Reply:
I have found reference to one bucket name in some test cases, using that as an output path, Let's run the test cases and see if it works.